src/components/HTMLEngineProvider/applyStrikethrough/index.js

```diff
@@ -0,0 +1,5 @@
+function applyStrikethrough(html) {
+    return html;
```

Comment:
Shouldn't we remove the styles which apply strike-through for web and unify the approach across platforms

Reply:
If we want to remove the style, that means we need to remove it from all of it's parent and that would affect other components.

Reply:
@youssef-lr your thoughts?

Reply:
The chat message is inside this `OfflineWithFeedback`, and if we don't want to have the strikethrough styles, then we need to pass the `pendingAction` props only if it's a delete action.
https://github.com/Expensify/App/blob/0763be562293a1b9e3782eaeee0843a7f805e38d/src/pages/home/report/ReportActionItem.js#L255-L279

Next, we also need to give the style manually to the non-HTML text here
https://github.com/Expensify/App/blob/0763be562293a1b9e3782eaeee0843a7f805e38d/src/pages/home/report/ReportActionItemFragment.js#L120-L137

This is assuming that all components inside the `OfflineWithFeedback` does not need the strikethrough style except `ReportActionItemFragment`.

Reply:
@youssef-lr bump thoughts here?

Reply:
>the web will remain untouched here as we're only adding them in index.native.js?

Yes

> Shouldn't we remove the styles which apply strike-through for the web and unify the approach across platforms?

As of now strike through styles for deleting messages are applied through styles on web, now we are handling native using `<del>` tag. Instead of mixing up two solutions one for the web & one for native shouldn't we unify?

Reply:
I don't really have a good solution, but are we fine with this?

Reply:
I'm fine with as is now, let me know if you differ @youssef-lr